tests/test_search_requests.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from cpr_data_access.models.search import SearchRequestBody
+from cpr_data_access.vespa import _build_yql, sanitize
+
+
+def test_whether_an_empty_query_string_raises_a_valueerror():
+    with pytest.raises(ValueError):
+        SearchRequestBody(query_string="")
+
+
+def test_whether_valid_year_ranges_are_accepted():
+    request = SearchRequestBody(query_string="test", year_range=(None, 2000))
+    assert isinstance(request, SearchRequestBody)
+
+    request = SearchRequestBody(query_string="test", year_range=(2000, None))
+    assert isinstance(request, SearchRequestBody)
+
+
+def test_whether_an_invalid_year_range_ranges_raises_a_valueerror():
+    with pytest.raises(ValueError):
+        SearchRequestBody(query_string="test", year_range=(2023, 2000))
+
+
+def test_whether_valid_sort_fields_are_accepted():
+    valid_sort_fields = ["date", "name"]
+    for field in valid_sort_fields:
+        request = SearchRequestBody(query_string="test", sort_by=field)
+        assert isinstance(request, SearchRequestBody)
```

Comment:
can I suggest parameterising these tests

```suggestion
@pytest.mark.parametrize("field", ["date", "name"])
def test_whether_valid_sort_fields_are_accepted(field):
    request = SearchRequestBody(query_string="test", sort_by=field)
    assert isinstance(request, SearchRequestBody)
```

Reply:
Great suggestion